agent/stats/queue_test.go

```diff
@@ -21,7 +21,6 @@ import (
 	"time"
```

Comment:
Most of the diff here is changing the tests to use the testify 'require' package rather than doing the comparisons ourselves.